adapters/adform/adform.go

```diff
@@ -586,7 +585,10 @@ func toOpenRtbBidResponse(adformBids []*adformBid, r *openrtb.BidRequest) *adapt
 		}
 
 		bidResponse.Bids = append(bidResponse.Bids, &adapters.TypedBid{Bid: &openRtbBid, BidType: openrtb_ext.BidTypeBanner})
+		currency = bid.Currency
```

Comment:
It might be useful to add a check to make sure we don't get bids back for a request that are associated with different currencies. Because in the very unlikely case that that might happen, there would be no way to know about it (except maybe from a user).

Reply:
If this check is about validating that bids match what we add to rcur, then we have it in our ad server